ansible_3par_docker_plugin/tasks/configure_doryd_service.yml

```diff
@@ -14,3 +14,34 @@
       daemon_reload: yes
       enabled: yes
       name: doryd.service
+
+  - name: collect facts about system services
+    service_facts:
+    register: services_state
+
+  - name: If the service failed to start, use another kubeconfig
+    ini_file:
+      path: /etc/systemd/system/doryd.service
+      section: Service
+      option: ExecStart
+      value: /usr/libexec/kubernetes/kubelet-plugins/volume/exec/hpe.com~hpe/doryd /etc/origin/master/admin.kubeconfig  hpe.com
+    when: services_state['ansible_facts']['services']['doryd.service']['state'] == "stopped"
```

Comment:
Can there be a possible race condition between the two attempts to start the service?
E.g. the admin.conf, is good and the service is about to come up but the state is still "stopped" so the attempt to start service with admin.kubeconfig is also made. Is some sleep or wait required between the two attempts? Please check.